src/shared/components/activity/activity-banner/activity-banner.component.scss

```diff
@@ -1,40 +1,31 @@
-.caseLocked {
-    margin-top: 4px;
-    height: 40px;
-    -webkit-filter: blur(0);
-    filter: blur(0);
+.activity-banner {
+  margin-top: 4px;
+  height: 40px;
+  background-color: #912b88;
+  &.case-locked {
     background-color: #e72626;
-}
-
-.someoneViewing {
-    margin-top: 4px;
-    height: 40px;
-    -webkit-filter: blur(0);
-    filter: blur(0);
-    background-color: #912b88;
-}
+  }
 
-.bannerIcon{
+  .banner-icon {
     float: left;
     color: #FFFFFF;
     padding-left: 9px;
     position: relative;
     top: 50%;
     transform: translateY(-40%);
-}
-
-.bannerText {
+  }
+  
```

Comment:
Did you run `yarn lint`? Extra white space here.

Reply:
No lint issues shown. I think it's because this is a `.scss` file and it only really cares about extra whitespace in `.ts` files...?

Reply:
Possible.